libs/application/core/src/lib/slugUtil.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  ApplicationConfigurations,
+  ApplicationTypes,
+} from '../types/ApplicationTypes'
+
+export const getTypeFromSlug = (slug: string) =>
+  Object.entries(ApplicationConfigurations).find(
+    ([_, item]) => item.slug === slug,
+  )?.[0] as ApplicationTypes | undefined
+
+export const getSlugFromType = (type: ApplicationTypes) =>
+  Object.entries(ApplicationConfigurations).find(([id]) => id === type)?.[0] as
+    | string
+    | undefined
```

Comment:
There is a bug here, you're returning `[0]` when you would want to return `[1].slug`

This could also use a similar refactor as the above function to end up like:
```typescript
for (const [key, value] of Object.entries(ApplicationConfigurations) {
  if (type === key) {
    return value.slug
  }
}

return undefined
```

If you find the type you're returning `[0]` which is the type, right?